docs/software/dns_servers.md

```diff
@@ -2,14 +2,114 @@
 
 ## DNS Server
 
-!!! warning "Documentation in progress..." 
+- [**Pi-hole - A DNS/Web server that will block ads for any device on your network**](#pi-hole-a-dnsweb-server-that-will-block-ads-for-any-device-on-your-network)
 
-    The links redirect to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+## Dynamic DNS Clients
 
-- [Pi-hole - A DNS/Web server that will block ads for any device on your network.](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=20#p174)  
+- [**No-IP - Dynamic DNS update client**](#no-ip-dynamic-dns-update-client)
 
-## Dynamic DNS Clients
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
+    ```
+    and selected _DietPi-Software_. or launch directly the tool:
+    ```
+    dietpi-software
+    ```
+
+    Choose **Software optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
 
-- [No-IP - Website URL Address](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p58)  
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+## Pi-hole - A DNS/Web server that will block ads for any device on your network
```

Comment:
"A [DNS sinkhole](https://wikipedia.org/wiki/DNS_sinkhole) with web interface that..."

Reply:
Changed to "Network-wide Ad Blocking" (like on the pi-hole homepage).